template-engine/chapter.md

```diff
@@ -776,12 +778,15 @@ which will mean that our compiled Python function will have this line:
 append_result('hello')
 ```
 
-which will add the string `hello` to the rendered output of the template.  The
+which will add the string `hello` to the rendered output of the template.  With
 multiple levels of abstraction here, it's important to keep them straight as we
-read the code.
+read the code. ((FIXME: Query: This is kind of an odd sentence at this point -- 
+it's a bit discouraging because the author doesn't offer any advice as to how to
+keep them straight. If it's just meant to be an acknowledgment that this is tricky,
+perhaps reword to "it's difficult to keep them straight"? --ARB))
```

Comment:
Maybe we change to, "We have multiple levels of abstraction here which can be difficult to keep straight.  The compiler uses `buffered.append("'hello'")`, which creates `append_result('hello')` in the compiled Python function, which when run, appends `hello` to the template result."   Not sure how much to spell it out I guess....